components/deep_sleep.rst

```diff
@@ -21,7 +21,9 @@ to wake up on any RTC pin (``GPIO0``, ``GPIO2``, ``GPIO4``, ``GPIO12``, ``GPIO13
 ``GPIO15``, ``GPIO25``, ``GPIO26``, ``GPIO27``, ``GPIO32``, ``GPIO39``).
 
 While in deep sleep mode, the node will not do any work and not respond to any network traffic,
-even Over The Air updates.
+even Over The Air updates. If the device's entities are appearing as **Unavailable** while your device is actively
+sleeping, this component was likely added after the device was added to Home Assistant. To prevent this behavior,
+you can remove and re-add the device within ESPHome.
```

Comment:
This should be HA, not esphome.

Reply:
https://github.com/esphome/esphome-docs/pull/3650